catalog/designers/degarismstudio/bio.html

```diff
@@ -0,0 +1,2 @@
+<p>Degarism Studio, established by Deni Anggara in 2015, is an independent design studio focused on typography, type, and editorial design—known for fonts such as Alliance and Biotif.</p>
```

Comment:
"focused on typography, type," sounds redundant. Could it be just typography?

Reply:
I've changed it to speed up the process. Please feel free to update it later if you think it needs to be restored.